css/css-ui/appearance-auto-001.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: appearance: auto</title>
+<link rel="help" href="https://drafts.csswg.org/css-ui-4/#appearance-switching">
+<meta name="assert" content="auto is supported.">
+<link rel="match" href="appearance-auto-ref.html">
+<style>
+ #container { width: 500px; }
+ #container > * { -webkit-appearance: none; -webkit-appearance: auto; }
```

Comment:
Same comment as I made in [an other pull request](https://github.com/web-platform-tests/wpt/pull/14993#discussion_r250016683): the prefixed version is mandated by spec, so it's ok to test with it, but we should not **only** test with it.

Reply:
I wrote a python script to generate the -webkit- prefixed reftests.